src/OnboardingSPA/components/ThemeContextProvider/index.js

```diff
@@ -0,0 +1,21 @@
+import { useState, createContext } from '@wordpress/element';
+
+const ThemeContext = createContext();
+
+const ThemeProvider = ( { children } ) => {
+	const [ theme, setTheme ] = useState( 'dark' );
+
+	const toggleTheme = () => {
+		setTheme( ( prevTheme ) =>
+			prevTheme === 'dark' ? 'light' : 'dark'
```

Comment:
Should we have constants? for `dark` and `light` cause I see the use case a lot.
I remember we had `Background` earlier and one day that was changed to `Base`